### PR TITLE
Allow toggling Universal SSL on a CNAME zone

### PR DESCRIPTION
**Pull request by stefanor**
This re-starts the verification process, if it had timed out.

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: pycloudflare/models.py [hunk 1]
```diff
@@ -197,6 +197,11 @@ def purge_cache(self, files=None, tags=None):
     def get_ssl_verification_info(self):
         return self._service.get_ssl_verification_info(self.id)
 
+    def re_verify(self):
+        for state in (False, True):
```
Comment (jamesodo) [inline pycloudflare/models.py]: This looks like something that should have an explanatory comment.  The "'Toggle Universal SSL off and on, to trigger re-verification'" (the help for the management commands) seems good enough.

Comment (stefanor) [reply pycloudflare/models.py]: Yeah. Also document the side-effect that SSL gets enabled.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Pull request merged**